.github/workflows/ci.yml

```diff
@@ -542,12 +542,26 @@ jobs:
           API: ON, CMAKE_FLAGS: "-DRYML_DEV=OFF -DRYML_BUILD_API=ON -DRYML_API_TESTS=ON -DRYML_API_BENCHMARKS=OFF"}
     steps:
       - {name: checkout, uses: actions/checkout@v2, with: {submodules: recursive}}
+      - {name: python3, uses: actions/setup-python@v2, with: {python-version: 3.7}}
       - {name: install requirements, run: source .github/reqs.sh && c4_install_test_requirements $OS}
       - {name: show info, run: source .github/setenv.sh && c4_show_info && which swig }
       - name: api-shared64-configure---------------------------------------------------
         run: source .github/setenv.sh && c4_cfg_test shared64
-      - {name: api-shared64-python-build, run: source .github/setenv.sh && c4_build_target shared64 ryml-api-python3}
-      - {name: api-shared64-python-test, run: source .github/setenv.sh && c4_build_target shared64 ryml-api-test-python3}
+      - name: api-shared64-python-build
+        run: |
+            python --version
+            python -mvenv env
+            source .github/setenv.sh
+            source env/bin/activate
```

Comment:
This is causing problems with the windows builds: https://github.com/biojppm/rapidyaml/pull/102/checks?check_run_id=1696849360#step:7:35

Reply:
Fixed